Cargo.toml

```diff
@@ -16,3 +16,4 @@ dunce = "1.0.1"
 glob = "0.3.0"
 serde = { version = "1.0.123", features = ["derive"] }
 toml = "0.5.8"
+current_platform = "0.2.0"
```

Comment:
can you add a new line there? most editors should be configurable to insert/remove trailing new lines on save. also should be sorted alphabetically (`cargo add` does that now, maybe you need to update `cargo-edit`)

Reply:
It looks like there's now two newlines, instead of the standard single trailing newline. :disappointed: